genienlp/ned/bootleg.py

```diff
@@ -173,21 +132,16 @@ def extract_mentions(self, input_path):
             verbose=False,
         )
 
-    def pad_values(self, tokens, max_size, pad_id):
-        if len(tokens) > max_size:
-            tokens = tokens[:max_size]
-        else:
-            tokens += [pad_id] * (max_size - len(tokens))
-        return tokens
-
     def disambiguate_mentions(self, config_args):
         run_model(self.args.bootleg_dump_mode, config_args)
 
     def post_process_bootleg_types(self, title):
```

Comment:
Why is the initialization for type postprocessing done in the parent class, but type postprocessing is only used in Bootleg?

Reply:
Oracle uses `wiki2normalized_type` to normalize types too. Except only uses the first half of this function.  I have defined a base `normalize_types` method that other classes would inherit in a new PR I'm working on. That would solve this confusion.